istioctl/cmd/root.go

```diff
@@ -350,18 +348,6 @@ func softGraduatedCmd(cmd *cobra.Command) *cobra.Command {
 	return &newCmd
 }
 
-// graduatedCmd is used for commands that have graduated and should not work if invoked the old way.
-func graduatedCmd(name string) *cobra.Command {
```

Comment:
@esnible I like this function and want to keep it but linter says `unused func`. can we use it for any other command that are graduated ?

Reply:
You **could** shut the linter up with `var _ = graduatedCmd` but that seems abusive.  If we need it again we can recreate it.